worker/dbaccessor/worker_integration_test.go

```diff
@@ -93,30 +93,79 @@ func (s *integrationSuite) TearDownSuite(c *gc.C) {
 }
 
 func (s *integrationSuite) TestWorkerAccessingControllerDB(c *gc.C) {
-	db, err := s.dbGettter.GetDB(coredatabase.ControllerNS)
+	db, err := s.dbManager.GetDB(coredatabase.ControllerNS)
 	c.Assert(err, jc.ErrorIsNil)
 	c.Assert(db, gc.NotNil)
 }
 
 func (s *integrationSuite) TestWorkerAccessingUnknownDB(c *gc.C) {
-	_, err := s.dbGettter.GetDB("foo")
+	_, err := s.dbManager.GetDB("foo")
 	c.Assert(err, gc.ErrorMatches, `.*namespace "foo" not found`)
 }
 
 func (s *integrationSuite) TestWorkerAccessingKnownDB(c *gc.C) {
-	db, err := s.dbGettter.GetDB(coredatabase.ControllerNS)
+	db, err := s.dbManager.GetDB(coredatabase.ControllerNS)
 	c.Assert(err, jc.ErrorIsNil)
 	err = db.Txn(context.TODO(), func(ctx context.Context, tx *sql.Tx) error {
 		_, err := tx.ExecContext(ctx, `INSERT INTO model_list (uuid) VALUES ("bar")`)
 		return err
 	})
 	c.Assert(err, jc.ErrorIsNil)
 
-	db, err = s.dbGettter.GetDB("bar")
+	db, err = s.dbManager.GetDB("bar")
 	c.Assert(err, jc.ErrorIsNil)
 	c.Assert(db, gc.NotNil)
 }
 
+func (s *integrationSuite) TestWorkerDeletingControllerDB(c *gc.C) {
+	err := s.dbManager.DeleteDB(coredatabase.ControllerNS)
+	c.Assert(err, gc.ErrorMatches, `.*cannot close controller database`)
+}
+
+func (s *integrationSuite) TestWorkerDeletingUnknownDB(c *gc.C) {
```

Comment:
This is not deleting the DB.

Reply:
Officially not, but it should. For now, there is a note about this.

Reply:
I mean that you're calling `GetDB` rather than `DeleteDB`.

Reply:
Ah I get you...